testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -403,8 +403,7 @@ def new_tenant_docker_client(self, name, tenant):
 class DockerComposeCompatibilitySetup(DockerComposeNamespace):
     def __init__(self, name, tag, enterprise=False):
         self._enterprise = enterprise
-        self.COMPAT_FILES_TEMPLATE[0] = self.COMPAT_FILES_TEMPLATE[0].format(tag=tag)
-        extra_files = self.COMPAT_FILES_TEMPLATE
+        extra_files = [self.COMPAT_FILES_TEMPLATE[0].format(tag=tag)]
```

Comment:
`self.COMPAT_FILES_TEMPLATE` should not be an array, just a (format) string.

So I suggest we rename above to `COMPAT_FILES_TEMPLATE = COMPAT_FILES_ROOT + "/docker-compose.compat-{tag}.yml"` and here:
```suggestion
        extra_files = [self.COMPAT_FILES_TEMPLATE.format(tag=tag)]
```

Reply:
sure, fixed. (I wanted to stay aligned with the res of the class variables, but it is OK)